package.json

```diff
@@ -32,12 +39,11 @@
     "postcss": "8.4.14",
     "postcss-loader": "7.0.0",
     "postcss-nesting": "10.1.8",
-    "prettier": "2.6.2",
-    "pretty-quick": "3.1.3",
     "style-loader": "3.3.1",
     "svgo-loader": "3.0.0",
     "ts-jest": "27.1.4",
     "ts-loader": "9.3.0",
+    "ts-standard": "^11.0.0",
```

Comment:
Please remove the `^` ... the BCP is to pick a particular version and not allow upgrades.